doc/source/catalog_generation.rst

```diff
@@ -436,6 +530,241 @@ good and the other contaminated.
+    | sky_centroid_icrs      | RA and DEC     | Equatorial coordinates in degrees                    |
+    +------------------------+----------------+------------------------------------------------------+
+    | source_sum             | FluxIso        | Sum of the unmasked data within the source segment   |
+    +------------------------+----------------+------------------------------------------------------+
+    | source_sum_err         | FluxIsoErr     | Uncertainty of FluxIso, propagated from input array  |
+    +------------------------+----------------+------------------------------------------------------+
+    | xcentroid              | X-Centroid     | X-coordinate of the centroid in the source segment   |
+    +------------------------+----------------+------------------------------------------------------+
+    | ycentroid              | Y-Centroid     | Y-coordinate of the centroid in the source segment   |
+    +------------------------+----------------+------------------------------------------------------+
+
+
+4.2: Aperture Photometry Measurements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The aperture photometry measurements included with the segmentation algorithm use the same configuration
+variable values and literally follow the same steps as what is done for the point algorithm as
+documented in Sections 2.2 - 2.4.  The fundamental difference between the point and segment computations is 
+the source position list used for the measurements.
+
+5: The Output Segment Catalogs
+------------------------------
+The metadata for the catalogs, both total detection and filter, as discussed in Sections 3.1 and 3.2, 
+is pre-dominantly the same.  The differences arise with respect to the specific columns present in the
+catalog.  The naming convention for the catalogs is also the same except the filter name is replaced 
+by the literal *total* for the total detection catalog:
+<TELESCOPE>_<PROPOSAL ID>_<OBSERVATION SET ID>_<INSTRUMENT>_<DETECTOR>_total_<DATASET NAME>_<CATALOG TYPE>.ecsv
+where CATALOG TYPE is either *point-cat* or *segment-cat*.
+Using the same example from Section 3.1, the resulting auto-generated segment total detection catalog 
+filename will be:
+
+* hst_98765_43_acs_wfc_total_j65c43_segment-cat.ecsv
+
+and the filter catalog filename will be:
+
+* hst_98765_43_acs_wfc_f606w_j65c43_segment-cat.ecsv
+
+5.1: Total Detection Segment Catalog 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The multi-filter detection level (aka total) catalog contains the fundamental position measurements of 
+the detected source: ID, X-Centroid, Y-Centroid, RA, and DEC, supplemented by some of the 
+aperture photometry measurements from *each* of the filter catalogs (ABMAG of the outer aperture, Concentration 
+Index, and Flags).  Effectively, the output Total Detection Segment Catalog is a distilled version of all of 
+the Filter Segment Catalogs.  
+
+5.2: Filter Segment Catalog 
```

Comment:
"Comparison to HLA Catalog" may be a more obvious section name instead.

Reply:
OK. Done